README.md

```diff
@@ -80,6 +80,24 @@ In this case we add a database URL for the dev and test environments.
 This means that if you run `lein repl`, the dev database will be used,
 and if you run `lein test`, the test database will be used.
 
+So that profiles you define in `profiles.clj` are merged into, rather than
+replacing profiles defined in `project.clj`, a composite profile can be
+created in `project.clj`:
+
+```clojure
+:profiles {:dev [:project/dev :profiles/dev]
+           :test [:project/test :profiles/test]
+           ;; only edit :profiles/* in profiles.clj
+           :profiles/dev  {}
+           :profiles/test {}
+           :project/dev {:source-paths ["src" "tool-src"]
+                         :dependencies [[midje "1.6.3"]]
+                         :plugins [[lein-auto "0.1.3"]]}
+           :project/test {}}
+```
+
+And then using the `:profiles/dev` key in your `profiles.clj`.
```

Comment:
Should this be:

    And then use the `:profiles/dev` key in your `projects.clj`.

?